webapp/src/filelist.tsx

```diff
@@ -99,7 +103,9 @@ export class FileList extends data.Component<ISettingsProps, FileListState> {
                     hasDelete={deleteFiles && !/^(main\.ts|pxt\.json)$/.test(file.name)}
                     className={(currentFile == file ? "active " : "") + (pkg.isTopLevel() ? "" : "nested ") + "item"}
                 >
-                    {file.name} {meta.isSaved ? "" : "*"}
+                    {file.name}
+                    {showStar ? "*" : ""}
+                    {meta.isGitModified ? " ↑" : ""}
```

Comment:
We should probably use an icon and not the unicode character. Semantic has a lot of arrows.

Reply:
we tried adding an icon (down arrow) but semantic and the menu CSS insists on docking the icon to the right of the view. Visually, this puts the arrow all the way to the right which is near invisible. Thus keeping it as unicode text right next to the file name.